app/models/email_template.rb

```diff
@@ -25,6 +27,14 @@ def self.search params
   end
 
   def users
+    projects, mentors_for_warning_emails, mentees_for_warning_emails = Project.all, [], []
```

Comment:
I think it would be best to move it to Project's model as we don't want `users` to get messy.